dev/TEAL_opcodes.md

```diff
@@ -668,6 +686,22 @@ See `bnz` for details on how branches work. `b` always jumps to the offset.
 - selects one of two values based on top-of-stack: A, B, C -> (if C != 0 then B else A)
 - LogicSigVersion >= 3
 
+## cover n
+
+- Opcode: 0x4e {uint8 depth}
+- Pops: *... stack*, any
+- Pushes: any
+- remove top of stack, and place it deeper in the stack such that N elements are above it
```

Comment:
What happens if N=0?
What happens if the stack has less than N+1 elements?

Reply:
n=0 does nothing
n too big = FAIL